tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -326,13 +326,14 @@ Check that superseded warning link takes user to superseding-publication data-ca
 
     user clicks element    testid:superseded-by-link
 
-    user waits until page contains    Browse our open data    %{WAIT_SMALL}
+    user waits until h1 is visible    Browse our open data
 
     user waits until page contains    ${PUBLICATION_NAME_SUPERSEDE}    %{WAIT_SMALL}
+    user checks page contains radio    ${RELEASE_NAME_SUPERSEDE}
 
-    user checks page does not contain    ${PUBLICATION_NAME_ARCHIVE}
-
-    user checks page for details dropdown    ${PUBLICATION_NAME_ARCHIVE}
```

Comment:
I've removed

```
user checks page for details dropdown    ${PUBLICATION_NAME_ARCHIVE}
```

because I can't see the details section which it's intending to check.